portal/views/patients.py

```diff
@@ -164,6 +165,20 @@ def session_report(subject_id, instrument_id, authored_date):
         authored_date=authored_date)
 
 
+@patients.route(
+    '/longitudinal-report/<int:subject_id>/<instrument_id>')
```

Comment:
RESTful design would suggest the patient id should be immediately following /patients.  
```suggestion
    '/<int:subject_id>/longitudinal-report/<instrument_id>')
```

Reply:
thank you, fixed